src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -113,6 +114,29 @@
       constants/content-type-system-message-mutual-event-sent
       [system-message-contact-request message-data :contact-request])))
 
+(defn bridge-message-content
+  [{:keys [bridge-message timestamp]}]
+  (let [{:keys [user-avatar user-name bridge-name content]} bridge-message]
+    [rn/view
+     {:style {:flex-direction     :row
+              :padding-horizontal 12
+              :padding-top        4}}
+     [fast-image/fast-image
+      {:source {:uri user-avatar}
+       :style  {:width         32
+                :margin-top    4
+                :border-radius 16
+                :height        32}}]
+     [rn/view {:margin-left 8 :flex 1}
+      [quo/author
+       {:primary-name   (str user-name)
+        :short-chat-key (str "Bridged from " bridge-name)
```

Comment:
maybe you want a translation key here? not the end of the world, but just for consistency

Reply:
we're not sure if we need this, we don't have designs I guess yet?

Reply:
so the entire implementation is kinda of temporary?

Reply:
oh yes, I guess so, though we are pretty certain `Briged from discord` will be there, but no strong opinion on my side, worse things happen at sea :)